src/libcrun/linux.c

```diff
@@ -4716,6 +4716,7 @@ libcrun_run_linux_container (libcrun_container_t *container, container_entrypoin
   size_t i;
   int ret;
 
+  libcrun_debug ("Running linux container");
```

Comment:
More or less a placeholder and TODO to add more verbosity to `libcrun_run_linux_container`.